lemmy_utils/src/settings.rs

```diff
@@ -134,7 +134,7 @@ impl Settings {
       .federation
       .allowed_instances
       .split(',')
-      .map(|d| d.to_string())
+      .map(|d| d.to_string().trim)
```

Comment:
This broke the build I think, I have this fixed in another branch, you can remove it.

Reply:
Oops I didnt even mean to include this here. Fixed.